conformance/tests/overloads_basic.py

```diff
@@ -79,3 +83,58 @@ def func2(x: int) -> int:  # E[func2]: no implementation
 @overload
 def func2(x: str) -> str:
     ...
+
+
+# > Overload definitions within stub files, protocols, and abstract base classes
+# > are exempt from this check.
+class MyProto(Protocol):
+    @overload
+    def func3(self, x: int) -> int:
+        ...
+
+
+    @overload
+    def func3(self, x: str) -> str:
+        ...
+
+class MyAbstractBase(ABC):
+    @overload
```

Comment:
This is a non-abstract function in an abstract base class, so it must have an implementation. If you add a `@abstractmethod` decorator to each of the overloads, it should not require an implementation.

Reply:
The current PEP draft might not make this sufficiently clear.